shell_test.go

```diff
@@ -437,9 +436,7 @@ func TestStorageUploadWithOffSign(t *testing.T) {
 	is.Nil(err)
 
 	uts := s.GetUts()
-	//sessionId, err := s.StorageUploadOffSign(mhash, uts)
-	sessionId, err := s.StorageUploadOffSign(mhash, uts, UploadMode("custom"),
-		Hosts(`16Uiu2HAmJk2N7Fqa7CFSMxu6DS3taGVtH4dWkLxxZZuoLuwbSAjc,16Uiu2HAmJk2N7Fqa7CFSMxu6DS3taGVtH4dWkLxxZZuoLuwbSAjc`))
+	sessionId, err := s.StorageUploadOffSign(mhash, uts)
```

Comment:
when I run this locally, i get the following:
13:17:38.156 ERROR core/comma: session error: no host available retry queue is empty session.go:914

Reply:
@steveyeom this is normal :)

Reply:
@uchetron Production? you can try again. Hub must not be working properly at that time.

If it's on staging/testnet, please let Richard know.

Reply:
@taiyangc ..so, Steve mentioned that this is normal behavior, which is why I added the comment here.

Reply:
The error message is not related to go-btfs-api. But it can be related to testnet or btfs daemon. In other words, the error message is not related to this PR.

Reply:
Agreed it's not related to this PR. But this message still shows problems with the payment process and should be triaged by the team. If not sure who to ask, you can put in tech-sync.